torch/__init__.py

```diff
@@ -569,6 +569,23 @@ def get_deterministic_debug_mode() -> builtins.int:
     else:
         return 0
 
+def get_float32_matmul_precision() -> builtins.str:
```

Comment:
The getter here makes sense although it's a little funny because if someone does set the underlying variables independent of this then they may get a setting that doesn't reflect those values

Maybe we should just nix the getter for now?

Reply:
Would this be the same situation that @ptrblck and @ngimel were describing (e.g., `allow_tf32` not matching the precision setting)? I think if we decided what would happen if the variables were unified (would `allow_tf32 = True` set to `HIGH` or `MEDIUM`?) then the getter would not be ambiguous.

Reply:
Yep -- you're right. Following the above let's keep the getter and it just returns the last value set. No unification (in the future we probably want to support specific overrides of this global precision, too).